pkg/ebpf/events_processor.go

```diff
@@ -423,3 +473,8 @@ func CopyFileByPath(src, dst string) error {
 	}
 	return nil
 }
+
+type HookedSyscallData struct {
```

Comment:
Shouldn't it be higher in the file? Maybe in another file (so it would be exported)?